Cask

```diff
@@ -5,4 +5,6 @@
 
 (development
  (depends-on "noflet")
- (depends-on "helm"))
+ (depends-on "ag")
+ (depends-on "helm")
+ (depends-on "helm-ag"))
```

Comment:
I can delete this lines. But i don't understand how to test without this. For testing `projectile-ag` i need package `ag`. I run `script/emacs.sh` and then in *scratch* buffer execute `(require 'ag)`. Same with `helm-projectile-ag`. How to test without this lines?